src/Menu.php

```diff
@@ -123,14 +123,16 @@ public function addGroup($name, string $template = 'menugroup.html')
             $name = [$name];
         }
 
-        $title = $name['title'] ?? $name['text'] ?? $name['name'] ?? $name[0] ?? null;
+        $title = $name['title'] ?? $name[0] ?? null;
 
         if ($title !== null) {
             $group->template->set('title', $title);
         }
 
         if (isset($name['icon'])) {
             Icon::addTo($group, [$name['icon']], ['Icon'])->removeClass('item');
+        } elseif (isset($name['label'])) {
+            Label::addTo($group, [$name['label']], ['Icon'])->removeClass('item')->addClass('tiny');
```

Comment:
```suggestion
            Label::addTo($group, [$name['label']], ['Icon'])->removeClass('item');
```

for menu item the size is standard (class is unspecified) - ie. the font size is 12 px

for menu group, the size should not be smaller